### ISSUE TITLE
Always show TZ in display

### ISSUE THREAD

**Issue by tomlayg6** [BasicAirData/GPSLogger#104]
Ver 2.2.10 changed the default from UTC to Local Time.
The TZ offset appears in the display, but only if it is not UTC.
This is confusing; I would expect the TZ to be shown always.

**Comment by GrazianoCapelli** [comment]
We chosen to show the Timezone only when the Option _show the Time in local Timezone_ is active.
When the Timezone shifting is active, we always show the Offset.
This means that, if the Timezone of the User corresponds to UTC, the App will show `(+00:00)`.

As you rightly wrote, the Option is active by Default. This means that the User is _conscious_ when he deactivates it.
When the option is deactivated, the Time is the raw GPST time, that doesn't depend on Timezone.
This case We chosen to not show any information about the Timezone to remind the User that the Option is not active.

In short, if the Timezone shifting is active, it is visible; otherwise no.
I always found this implementation clean and self explaining.

**Comment by GrazianoCapelli** [comment]
I close this issue, because we consider good the current implementation.
